snapshot/snapshotmaintenance/snapshotmaintenance_test.go

```diff
@@ -0,0 +1,208 @@
+package snapshotmaintenance
```

Comment:
nit: do you need access to internals? can this be in `snapshotmaintenance_test` ?